pytket/docs/display.md

```diff
@@ -17,7 +17,7 @@ pip install pytket-offline-display
 
 ```{eval-rst}
 .. automodule:: pytket.circuit.display
```

Comment:
This is arguably outside the scope of this PR but it would be nice to have the `render_circuit_jupyter` and `view_browser` functions at the top of this page followed by the "example usage" snippet and finally the rest of the module.

These two functions are the primary ways to use the circuit renderer the rest of the classes and functions related to configuring the circuit renderer.

Happy to do this in a follow up if you like.

Reply:
As discussed, I have rearranged the content, but you may still want to edit it at some point soon to improve the content itself